### PR TITLE
Copy and paste from editable list summary

### PR DESCRIPTION
**Pull request by leonie-koch**
RISDEV-4028

### CODE REVIEW

File: frontend/src/components/ActiveCitationSummary.vue
Comment (Nadav-B) [inline frontend/src/components/ActiveCitationSummary.vue:1]: I love it, great job exporting this summary component out, reducing the code lines by reusing already existing and tested components.

File: frontend/src/domain/relatedDocumentation.ts
Comment (Nadav-B) [inline frontend/src/domain/relatedDocumentation.ts:1]: Happy to see this part out. A long string in favor of our variables limits our flexibility to customize styling to part of it.

File: frontend/src/components/ActiveCitationSummary.vue
Comment (Nadav-B) [reply frontend/src/components/ActiveCitationSummary.vue:1]: @leonie-koch question here: in our [code](https://github.com/digitalservicebund/ris-backend-service/commit/fdfa79707073a533a0f53bcff6607a7ca4e948bd#diff-76c4fc531deecc6de1fdbec98285ee7eaff245595026459cca75cbb7dd5f8c11) we once use test id in camel and once in underscore. 

`await page.getByTestId("listEntry-0").click()`

[Can we decide for the official guideline of the library](https://testing-library.com/docs/queries/bytestid/#api)?

Comment (leonie-koch) [reply frontend/src/components/ActiveCitationSummary.vue:1]: Absolutely! Haven't seen that, thanks for pointing out, i will change that 👍

**Pull request merged**